### ISSUE TITLE
Fix automated tests that were commented when switching to containers

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/ansible-role-mailhog#9]
When I switched over Travis tests to use Docker, I also commented out the tests I was using to ensure mailhog was running smoothly:

``` yaml
  # TODO - Fix the following tests so they run inside Docker.
  # Make sure mailhog is listening on ports 1025 and 8025.
  # - sudo netstat -taupen | grep '1025.*mailhog'
  # - sudo netstat -taupen | grep '8025.*mailhog'

  # Send an email via ssmtp.
  # - cat tests/message | ssmtp johndoe@example.com

  # Test retrieving messages via MailHog API.
  # - curl http://localhost:8025/api/v2/messages | grep id
```

I need to un-comment those tests and run them inside the container context so the automated tests verify Mailhog is running as it should post-role-run.

**Comment by geerlingguy** [comment]
Grr... leaving things turned off for now, as I could only reliably get some of the tests to work in CentOS 7. Something with Docker and the networking maybe :/

**Comment by geerlingguy** [comment]
Fixed. Had to use `sh -c` to wrap the command requiring a pipe/redirection:

```
  - 'sudo docker exec --tty "$(cat ${container_id})" env TERM=xterm sh -c "cat /etc/ansible/roles/role_under_test/tests/message | /opt/mailhog/mhsendmail johndoe@example.com"'
```

**Comment by geerlingguy** [comment]
(Fixed in one of the commits on #4)
